### PR TITLE
add metrics for delegator insert/delete cost

### PR DESCRIPTION
**Pull request by weiliu1031**
Signed-off-by: Wei Liu <wei.liu@zilliz.com>
/kind improvement

### CODE REVIEW

File: pkg/metrics/querynode_metrics.go [hunk 1]
```diff
@@ -47,6 +47,27 @@ var (
...
+	QueryNodeProcessCost = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: milvusNamespace,
+			Subsystem: typeutil.QueryNodeRole,
+			Name:      "process_insert_or_delete_ms",
+			Help:      "process insert or delete cost in ms",
+		}, []string{
+			nodeIDLabelName,
+			msgTypeLabelName,
+		})
```
Comment (congqixia) [inline pkg/metrics/querynode_metrics.go:59]: maybe use histogram instead of gauge is better here?

**Review by jiaoew1991 — COMMENTED** [review]
/lgtm

**Pull request merged**